freelist.go

```diff
@@ -313,13 +313,13 @@ func (f *freelist) write(p *common.Page) error {
 		p.SetCount(uint16(l))
 		var ids []common.Pgid
 		data := common.UnsafeAdd(unsafe.Pointer(p), unsafe.Sizeof(*p))
-		common.UnsafeSlice(unsafe.Pointer(&ids), data, l)
+		ids = unsafe.Slice((*common.Pgid)(data), l)
```

Comment:
minor comment: line 314 can be removed? same below.
```suggestion
		ids := unsafe.Slice((*common.Pgid)(data), l)
```